endoreg_db/serializers/_old/raw_pdf_meta_validation.py

```
@@ -94,7 +94,7 @@ def validate(self, data):
 
 
 from rest_framework import serializers
-from ..models import SensitiveMeta
+from ...models import SensitiveMeta
```

Ruff (0.8.2)

Redefinition of unused `SensitiveMeta` from line 4

(F811)